### PR TITLE
Fix #2570 security password dialog issue

### LINKED ISSUE #2570: Repeatedly create new password issue
**Actual Behaviour**

Every time we on the security (Access by password) we need to create new password even if there is an option for change password.

**Expected Behaviour**

It should ask or show dialog only for first time.

**Would you like to work on the issue?**

Yes.

### PR DESCRIPTION
**Pull request by sparrow007**
Fixed #2570 

Changes: [Check for password if it's already done then simply turn on security and if not the make new password setup.]

GIF of the change:

### CODE REVIEW

File: app/src/main/java/org/fossasia/phimpme/gallery/activities/SecurityActivity.java [hunk 1]
```diff
@@ -94,10 +94,16 @@ public void onCheckedChanged(CompoundButton buttonView, boolean isChecked) {
...
+                String password = SP.getString(getString(R.string.preference_password_value),"");
+                if (password.isEmpty() && isChecked)
                     setPasswordDialog();
+                else if (isChecked) {
+                    swActiveSecurity.setChecked(true);
+                    editor.putBoolean(getString(R.string.preference_use_password), true);
+                    editor.commit();
+                    toggleEnabledChild(true);
+                }
                 else {
```
Comment (yashk2000) [inline app/src/main/java/org/fossasia/phimpme/gallery/activities/SecurityActivity.java]: Shift this `else {` to the line above it. Make it `} else {`.

**Comment by abishekvashok** [comment]
Updates pls

**Comment by sparrow007** [comment]
@abishekvashok pls review new changes.

**Comment by yashk2000** [comment]
@sparrow007, I think that this issue has been already taken care of by #2269.

**Comment by sparrow007** [comment]
@yashk2000 ok then i am closing this request.

**Pull request closed without merging**